src/components/balance/PlmBalance.vue

```diff
@@ -85,46 +68,53 @@
             >
               {{ $t('balance.faucet') }}
             </button>
-
-            <!-- memo: duplicate component to avoid styling from breaking -->
-            <button
-              v-if="!isH160 && !isEvmDeposit"
-              :disabled="!canUnlockVestedTokens"
-              type="button"
-              class="transfer-button large-button"
-              @click="unlockVestedTokens"
-            >
-              {{ $t('balance.unlockVestedTokens') }}
-            </button>
-            <button
-              v-if="isEvmDeposit && !isH160"
-              type="button"
-              class="transfer-button large-button"
-              @click="openWithdrawalModal"
-            >
-              {{ $t('balance.withdrawEvm') }}
-            </button>
           </div>
+
+          <button
+            v-if="isEvmDeposit && !isH160"
+            type="button"
+            class="transfer-button"
+            @click="openWithdrawalModal"
+          >
+            {{ $t('balance.withdrawEvm') }}
+          </button>
         </div>
       </div>
     </div>
 
     <div class="tw-text-sm tw-flex tw-flex-col tw-gap-y-5">
       <div
         class="
-          tw-flex tw-justify-between tw-items-center tw-bg-blue-50
+          tw-flex tw-flex-col tw-justify-between tw-bg-blue-50
           dark:tw-bg-darkGray-700
           tw-rounded-lg tw-py-3 tw-px-4
         "
       >
-        <div>{{ $t('balance.vested') }}</div>
-        <div>
-          <p class="tw-font-bold tw-text-right">
+        <div class="tw-flex tw-justify-between">
+          <div>
+            {{ $t('balance.claimable') }}
+          </div>
+          <div class="tw-font-bold tw-text-right tw-fle">
             <span class="tw-text-2xl md:tw-text-xl xl:tw-text-2xl tw-leading-tight">
-              <format-balance :balance="accountData?.vested" />
+              <format-balance :balance="accountData?.vestedClaimable" />
             </span>
-          </p>
+          </div>
         </div>
+        <div class="tw-flex tw-justify-between">
+          <div>
+            {{ $t('balance.vested') }}
+          </div>
+          <div>
+            <p class="tw-font-bold tw-text-right">
+              <span class="tw-text-2xl md:tw-text-xl xl:tw-text-2xl tw-leading-tight">
+                <format-balance :balance="accountData?.vested" />
+              </span>
+            </p>
+          </div>
+        </div>
+        <Button :small="true" class="tw-self-end tw-mt-1" @click="showVestingInfo">
```

Comment:
![Screen Shot 2022-01-20 at 4 18 55 PM](https://user-images.githubusercontent.com/40356749/150367629-f8491432-9ba9-423e-b29b-4303dac2b8ba.png)

I like the staked view of the numbers. But the button position does not look very good. Can we move the `Info` button somewhere else? For example, next to the `Claimable` next or next to the faucet button and change the name of the button.